tool/travis.sh

```diff
@@ -8,8 +8,8 @@
 set -e
 
 # Install the bower and vulcanize.
-npm install -g bower
-npm install -g vulcanize
+# npm install -g bower
```

Comment:
Commenting out lgtm, but can you add a comment saying why? This will help the next person through :)